Makefile

```diff
@@ -0,0 +1,36 @@
+# Makefile
+
+all:
+	@echo ""
+	@echo "  install: install glide (assumes go is installed)"
+	@echo "  deps: grab dependencies using glide"
+	@echo "  test: test prebid-server (via validate.sh)"
+	@echo "  build: build prebid-server"
+	@echo "  image: build docker image"
+	@echo ""
+
+.PHONY: install deps test build image
+
+# install glide https://github.com/Masterminds/glide (assumes go is already installed)
+install:
+	curl https://glide.sh/get | sh
+
+# debs will clean out the vendor directory and use glide for a fresh install
```

Comment:
Good catch. I also should add testing individual adapters since that was brought up in slack a week or two ago. Will work on today.